migrations/v6.js

```diff
@@ -0,0 +1,249 @@
+import { describe, getCourse, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Box menu - v6.0.2 to v6.1.0', async () => {
+
+  // https://github.com/adaptlearning/adapt-contrib-boxMenu/compare/v6.0.2..v6.1.0
+
+  let menusWithHeaders;
+  const defaultTextAlignment = {
+    _title: '',
+    _body: '',
+    _instruction: ''
+  };
+
+  whereFromPlugin('Box menu - from v6.0.2', { name: 'adapt-contrib-boxMenu', version: '<6.1.0' });
+
+  whereContent('Box menu - where menus have _menuHeader', async (content) => {
+    const candidates = [getCourse(), ...content.filter(({ _type, _component }) => _type === 'menu' && (!_component || _component === 'boxMenu'))];
+    menusWithHeaders = candidates.filter(({ _boxMenu }) => _boxMenu?._menuHeader);
+    return menusWithHeaders.length;
+  });
```

Comment:
`[getCourse(), ...content.filter(({ _type, _component }) => _type === 'menu' && (!_component || _component === 'boxMenu'))];` this is really complicated for what it does.

Maybe something a bit easier to read?

```suggestion
  function getBoxMenus(content) {
    return content.filter(({ _type, _component }) => 
      (_type === 'menu' || _type === 'course') &&
      (!_component || _component === 'boxMenu'))
  }

  whereFromPlugin('Box menu - from v6.0.2', { name: 'adapt-contrib-boxMenu', version: '<6.1.0' });

  whereContent('Box menu - where menus have _menuHeader', async (content) => {
    menusWithHeaders = getBoxMenus(content).filter(({ _boxMenu }) => _boxMenu?._menuHeader);
    return menusWithHeaders.length;
  });
```